app/helpers/govuk_link_helper.rb

```diff
@@ -122,6 +122,19 @@ def extract_button_classes(inverse: false, secondary: false, warning: false)
   def brand
     Govuk::Components.brand
   end
+
+  def build_text(original, visually_hidden_prefix:, visually_hidden_suffix:, &block)
+    text = (block_given?) ? block.call : original
+
+    safe_join(
+      [
+        govuk_visually_hidden(visually_hidden_prefix),
+        text,
+        govuk_visually_hidden(visually_hidden_suffix),
+      ].compact,
+      " "
```

Comment:
This currently adds space between the visually hidden span and the text - I'll switch it to putting the space inside the visually hidden span, I think that's the right approach.